code/modules/hydroponics/grown.dm

```diff
@@ -73,10 +73,15 @@
 		var/reag_txt = ""
 		if(seed && P_analyzer.scan_mode == PLANT_SCANMODE_CHEMICALS)
 			msg += "<br><span class='info'>*Plant Reagents:*</span>"
+			var/chem_cap = 0
 			for(var/reagent_id in seed.reagents_add)
 				var/datum/reagent/R  = GLOB.chemical_reagents_list[reagent_id]
 				var/amt = reagents.get_reagent_amount(reagent_id)
-				reag_txt += "\n<span class='info'>- [R.name]: [amt]</span>"
+				chem_cap += seed.reagents_add[reagent_id]
+				if(chem_cap <= 1)
+					reag_txt += "\n<span class='info'>- [R.name]: [amt]</span>"
+				else
+					reag_txt += "\n<span class='warning'>- [R.name]: [amt] (OVER 100% CAPACITY)</span>"

```

Comment:
why is this here? the number "100%" is arbitrary; a 50% acid production+10% nutriment production will hit the chem cap for a 100 potency, 50u capacity plant, and 50%+10%=60%<100%

just delete this bit, it's gonna confuse people